blazor/pdfviewer-2/how-to/change-the-highlighted-color-of-the-text.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Change the highlight color of the text in Blazor SfPdfViewer Component | Syncfusion
+title: Change highlight color text in Blazor PDFViewer Component | Syncfusion
```

Comment:
Update control name like as our FT page.
<img width="348" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/44014283/38fcee2c-e4cf-4037-ba39-936281f6aa33">
If the title exceeds 70 characters, you may omit the word 'component' from the title of this file."

Reply:
Resolved